cohort_4/playbook/ACTIVITIES.md

```diff
@@ -689,6 +689,35 @@ Uniqueness is also important - no two people should be paired together again
 unless everyone has already been paired with each other. This will help members
 of the club all form bonds with each other.
 
+## Types of Pairing
+
+### Pair programming between beginner-advanced pairs
+
+Pair programming between beginner and advanced hackers is fairly similar to
+normal pair programming. The most benefit will come from the beginner driving
+and the advanced hacker navigating, so rotating is not as important a factor in
+these situations (hackers should rotate if their bordom becomes a blocking
+factor for them to continue working together, but optimize for the beginner
+driving and the advanced hacker navigating as much as possible).
+
+When pairing between beginner and advanced hackers, make sure each hacker
+understands what they are getting out of it:
+
+For the advanced hacker:
+
+- True mastery of a subject can only come once someone is at the point they can
+  explain it to someone. If a hacker can't navigate while pairing, they dont
+  have an indepth understanding of the topic.
+- This is practice in working with people who don't know as much
+  subject-specific information. Teaching this content to a beginning hacker is
+  practice for any future club leading activities.
```

Comment:
I'd suggest removing the "doing it in a club" context entirely – not all advanced coders will want to lead a club.